### PR TITLE
Added some missing repo files

### PR DESCRIPTION
**Pull request by makermelissa**
Added a setup.py file as well as adding some PyPI stuff to the Travis.yml file.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,20 +6,35 @@
...
+    skip_cleanup: true
+    overwrite: true
+    on:
+      tags: true
+  # TODO: Use 'travis encrypt --com -r adafruit/<repo slug>' to generate
+  #       the encrypted password for adafruit-travis. Paste result below.
+  - provider: pypi
+    user: adafruit-travis
+    password:
+      secure: DC9aEeXOKnkvA9tPIi/ePQIW0SHdxcvjrUuZpoC1BXtFPqpMh3l3/rblYPgvVsfP612g/4BjeMpVKJoAhTfChotmoLZfH7zXsRf4sWKYkvMfrLBth5fZSHqaELU9IBIXLs2740DBSM7h6HgN8lh0jiOmr2F/JUXilqMCmXMrozU/LskficUpC8exvnVX0+yM3Da5+Awp09mJixXrJkvxvB5VW01o4ewPxucXCSPpeVCCUpk1HYK1Qla9ZyOL+hwuZk5dzm1u4atw3IpbUQeEw4m8dpt+JAB+2439V9ekK5neUGben+a3a2WAmyvwo4sfF/EqtgF3048EWz4SqF60DE8G8BeMPGSs2gzfvULLXcyAwwgluSkfWk6pk0ULJc+mOE63vFODsONDIQs/zfIDu3fiI6TYMiYRh7u7NHIYpVHukGeC9q2sPTr/EeLxLOE0W+Npnf/+6wycmV4bjauhuS71EGkGIzV2/gE+iuqmoC8ileidCVnS+2UZj+tPR2yQEdUoyTvKa3KcRQsOpXGNvzqJtI1NnHPv5V8BfZXUrqBJrq55SCrGDdl6jZ3INSor+1ASs4Cv3iCsfF3P0a7dSOsBjDzH2SDLiupi1+XLHR7v8MmGsF6o1CIc6/ML2kqD6r7wgIlF3OSXrA2JVs2piw/zMu9r6MyTYXenzVw5cyg=
```
Comment (kattni) [inline .travis.yml]: The secure hash must be generated on a repo-specific basis using both specific GitHub and PyPI credentials. It's not a common hash. The new `cookiecutter` adds the rest of the info you added, but does not include the hash as that is the final step to deploying to PyPI.

File: .travis.yml [hunk 2]
```diff
@@ -6,20 +6,35 @@
...
+  - "3.6"
 
 cache:
     pip: true
 
+# TODO: if deployment to PyPi is desired, change 'DEPLOY_PYPI' to "true",
+#       or remove the env block entirely and remove the condition in the
+#       deploy block.
+env:
+  - DEPLOY_PYPI="true"
```
Comment (kattni) [inline .travis.yml]: Until we add the correct secure hash, please leave this as `"false"`.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        'Topic :: Software Development :: Libraries',
+        'Topic :: System :: Hardware',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+    ],
+
+    # What does your project relate to?
+    keywords='adafruit ra8875 driver hardware micropython circuitpython',
```
Comment (kattni) [inline setup.py]: Please include more keywords. You put so much work into this - there's a lot more to it than simply its name! Include at least `display`, and anything else you find relevant. This is the set of keywords that is used when people are searching for a driver on PyPI.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    # You can just specify the packages manually here if your project is
+    # simple. Or you can use find_packages().
+    py_modules=['adafruit_ra8875'],
```
Comment (kattni) [inline setup.py]: `py_modules` must be changed to `packages` because this driver uses a package. The rest of the line will stay the same.

**Comment by kattni** [comment]
@makermelissa Ping me when you are working on this and I can generate the correct hash for you, then we can get PyPI deployed through this pull request.

**Comment by makermelissa** [comment]
Hi Kattni, I'm ready for the hash.

**Comment by kattni** [comment]
@makermelissa Excellent! Hopefully this works, I've never generated a hash without having cloned the repo first (it shouldn't be necessary the way we do it, but I've simply never tried).
```
AF+E8LEvOOLe66/++iw1plWdKmRH39gwdmzMzIJZlUdMGmbLYi0rFg+yC4+Xfvm9ZumG3aTLbWdzyvBHkB3H3/fT4YZf5mElxTEwGeYcOIburFQUNm0GqgXzKjfwvgH6lTNSWo2zibALrlytTnDPNvUfqyhAGvJBp2JgDUVS4kY1LjvzQSFCWcGNQ3F9kxsWeLHv34lTkjXQSrQNMLotjD4jVp6LAZMTbDtJKCGgfrYfOP60InVNWMuqVDB0DVFk+6ry8QbnTif9wsTiCoIAY7H25xFcJ/laFNRbx5nj3n+XVApM5EebJk+Ig5j/akSBa+MlmGvSTcHF+hP89+fRwhyJtu4qo2d1wu8D29lXs9VDVsw0tsix0CLFzEQsurUqaHfsZGSKnW9f/2v4+tyQyo4e3Hkvumm2Bu5xvREwnaAiSRw2Iavm9noGF/AnH5593AmKdkRtAWq0UrEDGKjp5enlZwhq1pkzM+7isOKDgsVOe2RnI1SXSgKq43qU2/xVCfhh5W/MBKLgLaTTH6iaXfSsZYwekuWIhdjE2IUf4rnbqjGhJIC/mObHAgsrrtWbKqDHuQg7xMV0MlSS5b1B6KuQrxouVz2BC/a9RcOm9k0EBHfqVEi1EJBfgJkYEeoCiKzwT+O2zhN5rgGC67NpVMKWfCl9sRT0gpZ9JE1H7OY=
```

File: .travis.yml (hunk 2 above)
Comment (kattni) [reply .travis.yml]: Leave this as `"true"` since we're adding the hash.

**Comment by kattni** [comment]
@makermelissa Please do a release titled something like "Now on PyPI". Then we'll check Travis to make sure it deployed properly.

**Pull request merged**